components/Player/NotesTable.tsx

```diff
@@ -1,23 +1,97 @@
 import { Notes } from "@prisma/client";
+import Button from "components/Button";
+import React from "react";
+
+const Note: React.FunctionComponent<{ note: Notes }> = ({
+  // eslint-disable-next-line camelcase
+  note: { created_at, content },
```

Comment:
I think should be able to get all fields of notes here so that you can fill it into each entry so:    
```
    id: string;
    author: number;
    created_at: Date;
    content: string;
    type: notetype;
```

Reply:
not building bc of unused parameters if I try to add it in so I added in a comment to remind us to add it in later!